### PR TITLE
Adding notes

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
completed till adding notes modal

### CODE REVIEW

File: src/components/sidebar/SideBar.jsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                width="25"
+                height="25"
+                preserveAspectRatio="xMidYMid meet"
+                viewBox="0 0 32 32"
+              >
+                <path
+                  fill="#020202"
+                  d="M16.612 2.214a1.01 1.01 0 0 0-1.242 0L1 13.419l1.243 1.572L4 13.621V26a2.004 2.004 0 0 0 2 2h20a2.004 2.004 0 0 0 2-2V13.63L29.757 15L31 13.428ZM18 26h-4v-8h4Zm2 0v-8a2.002 2.002 0 0 0-2-2h-4a2.002 2.002 0 0 0-2 2v8H6V12.062l10-7.79l10 7.8V26Z"
+                ></path>
+              </svg>
```
Comment (JowelTisso) [inline src/components/sidebar/SideBar.jsx:28]: You can prefer using react-icons instead of harcoded svgs

File: src/css/home.css [hunk 2]
```diff
@@ -80,7 +89,7 @@
     margin: 0;
 }
   
-.input-field {
+.input-search {
     width: 100%;
     border: 1px solid #E6E6E6;
     margin: 8px 0;
```
Comment (JowelTisso) [inline src/css/home.css:95]: * Prefer to use color variables

**Review by JowelTisso — APPROVED** [review]
* Clean code
* Rest Looks Good

File: src/context/NoteContext.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        isArchive:false,
+        isTrash:false
+    })
+    const [notes,setNotes]=useState([])
+    const changeHandler=(e)=>{
+        const name=e.target.name;
+        const value=e.target.value;
+        setNoteInput((prev)=>({...prev,[name]:value}))
+    }
+    const submitHandler=()=>{
```
Comment (shraddha-amber) [inline src/context/NoteContext.jsx:21]: remove empty handlers and console logs

**Review by shraddha-amber — APPROVED** [review]
* Try to write a more descriptive PR description
* And better commit messages
* Rest looks good!

File: src/css/home.css [hunk 4]
```diff
@@ -98,11 +109,11 @@
     border-radius: 6px;
 }
 
-.note-editor:hover{
+.note:hover{
     box-shadow: 0px 2px 2px rgba(154, 154, 154, 0.8);
```
Comment (Nikhil-Nirmal) [inline src/css/home.css:113]: Use color variables

**Review by Nikhil-Nirmal — APPROVED** [review]
Everything else looks good to merge

File: src/context/NoteContext.jsx (hunk 3 above)
Comment (mohammadkhizerkhan) [reply src/context/NoteContext.jsx:21]: sure thanks

File: src/css/home.css (hunk 2 above)
Comment (mohammadkhizerkhan) [reply src/css/home.css:95]: sure thanks

File: src/css/home.css (hunk 4 above)
Comment (mohammadkhizerkhan) [reply src/css/home.css:113]: sure thanks

**Pull request merged**